module_uninstall_check/wizards/wizard_module_uninstall_line.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class WizardModuleUninstallLine(models.TransientModel):
+    _name = 'wizard.module.uninstall.line'
+
+    TYPE_SELECTION = [
+        ('model', 'Model'),
+        ('field', 'Field'),
+    ]
+
+    wizard_id = fields.Many2one(
+        comodel_name='wizard.module.uninstall', required=True)
+
+    type = fields.Selection(
+        selection=TYPE_SELECTION, required=True)
+
+    model_id = fields.Many2one(comodel_name='ir.model', readonly=True)
+
+    model_name = fields.Char(
+        string='Model Name', related='model_id.model', readonly=True)
+
+    model_row_qty = fields.Integer(
+        string='Row Quantity', compute='_compute_model_row_qty')
+
+    field_id = fields.Many2one(comodel_name='ir.model.fields', readonly=True)
+
+    field_name = fields.Char(
+        string='Field Name', related='field_id.name', readonly=True)
+
+    field_ttype = fields.Selection(
+        string='Field Type', related='field_id.ttype', readonly=True)
+
+    field_model_name = fields.Char(
+        string='Field Model Name', related='field_id.model_id.model',
+        readonly=True)
+
+    @api.multi
+    @api.depends('model_id')
+    def _compute_model_row_qty(self):
```

Comment:
I think we could re-write in a more simple way like this:
```
for line in self.filtered(lambda x: x.line_type == 'model'):
    model_env = self.env[line.model_id.model]
    line.model_row_qty = model_env.search_count()  # Note: this will ignore inactive records. Not sure if it is meaningful to take them into account anyway.

```
I understand we might loose in term of performance but this should not be critical considering how often we will use this feature.